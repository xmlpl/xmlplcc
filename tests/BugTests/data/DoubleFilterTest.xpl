node[] main() {
  element e = <x><y a="0"/><y a="1"/></x>;

  e/y[@a]; "\n";
}
