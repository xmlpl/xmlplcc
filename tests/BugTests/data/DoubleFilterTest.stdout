<?xml version="1.0" encoding="UTF-8"?>
<y a="0"/><y a="1"/>
